build/crds/apps/apps_v1alpha1_edgeapplication.yaml

```diff
@@ -1,10 +1,9 @@
-
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.6.2
+    controller-gen.kubebuilder.io/version: v0.8.0
```

Comment:
Is it possible to upgrade the kubebuilder version in another PR? In this PR, let's focus on fixing the typo and comments.

Reply:
okay.

Reply:
But I need to revert the second commit, which ran `make generate verify`. Will the CI pass without this commit?

Reply:
Done.